[Bug] Unable to compile the project on Visual Studio 2026

## Description
Accomplish runs into a missing VS path error when tried to compile on Visual Studio 2026 and relies on Visual Studio 2022 Build Tools to compile and run properly.

## Steps to Reproduce
1. Clone the repository on VS 2026.
2. Run pnpm install and pnpm dev.
3. When pnpm dev is run, it fails due to missing VS 2022 Build Tools.

## Expected Behavior
The project should be updated to support VS 2026 Build Tools without having to install VS 2022 Build Tools.

## Temporary Workaround
Installing VS 2022 with C++ Workload / VS 2022 Build Tools mitigates the error and the project compiles and runs successfully.

## Environment
- OS: Windows 11
- Accomplish Version: 0.1.0

## Screenshots
<img width="1920" height="1020" alt="Image" src="https://github.com/user-attachments/assets/9cddc870-5cc5-4c4f-ab35-fc952317bfa5" />

## Terminal Output
```
**********************************************************************
** Visual Studio 2026 Developer PowerShell v18.3.0
** Copyright (c) 2025 Microsoft Corporation
**********************************************************************
PS C:\Users\joshu\Projects\accomplish> pnpm dev

> accomplish@0.1.0 predev C:\Users\joshu\Projects\accomplish
> node scripts/check-deps.cjs

✓ Dependencies in sync

> accomplish@0.1.0 dev C:\Users\joshu\Projects\accomplish
> pnpm -F @accomplish/desktop dev


> @accomplish/desktop@0.3.8 dev C:\Users\joshu\Projects\accomplish\apps\desktop
> node scripts/patch-electron-name.cjs && npx electron-rebuild -f && node -e "require('fs').rmSync('dist-electron',{recursive:true,force:true})" && vite

[patch-electron-name] Skipping on non-macOS platform
npm warn Unknown env config "msbuild-path". This will stop working in the next major version of npm.
npm warn Unknown env config "msvs-version". This will stop working in the next major version of npm.
npm warn Unknown env config "recursive". This will stop working in the next major version of npm.
| Building module: node-pty, Completed: 1
Error: Could not find any Visual Studio installation to use
    at VisualStudioFinder.fail (C:\Users\joshu\Projects\accomplish\node_modules\.pnpm\node-gyp@11.5.0\node_modules\node-gyp\lib\find-visualstudio.js:118:11)
    at VisualStudioFinder.findVisualStudio (C:\Users\joshu\Projects\accomplish\node_modules\.pnpm\node-gyp@11.5.0\node_modules\node-gyp\lib\find-visualstudio.js:74:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:103:5)
    at async createBuildDir (C:\Users\joshu\Projects\accomplish\node_modules\.pnpm\node-gyp@11.5.0\node_modules\node-gyp\lib\configure.js:112:18)
    at async process.<anonymous> (file:///C:/Users/joshu/Projects/accomplish/node_modules/.pnpm/@electron+rebuild@4.0.2/node_modules/@electron/rebuild/lib/module-type/node-gyp/worker.js:22:13)

× Rebuild Failed

An unhandled error occurred inside electron-rebuild
node-gyp failed to rebuild 'C:\Users\joshu\Projects\accomplish\node_modules\.pnpm\node-pty@1.1.0\node_modules\node-pty'

Error: node-gyp failed to rebuild 'C:\Users\joshu\Projects\accomplish\node_modules\.pnpm\node-pty@1.1.0\node_modules\node-pty'
    at ChildProcess.<anonymous> (file:///C:/Users/joshu/Projects/accomplish/node_modules/.pnpm/@electron+rebuild@4.0.2/node_modules/@electron/rebuild/lib/module-type/node-gyp/node-gyp.js:114:24)
    at ChildProcess.emit (node:events:508:28)
    at ChildProcess._handle.onexit (node:internal/child_process:294:12)
C:\Users\joshu\Projects\accomplish\apps\desktop:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @accomplish/desktop@0.3.8 dev: `node scripts/patch-electron-name.cjs && npx electron-rebuild -f && node -e "require('fs').rmSync('dist-electron',{recursive:true,force:true})" && vite`
Exit status 4294967295
 ELIFECYCLE  Command failed with exit code 4294967295.
```

--- a/.npmrc
+++ b/.npmrc
@@ -1,1 +1,0 @@
-msvs_version=2022

--- a/apps/desktop/scripts/dev.cjs
+++ b/apps/desktop/scripts/dev.cjs
@@ -64,11 +64,11 @@ function ensureNativeModules(commandEnv) {
 }
 
 function runElectronRebuild(commandEnv) {
-  // Pass --msvs_version=2022 so node-gyp selects VS 2022 Build Tools even
-  // when a newer Visual Studio (e.g. 2026) is the system default.
-  // This fixes compilation on machines that only have VS 2026 installed
-  // alongside the VS 2022 Build Tools workload. See GitHub issue #445.
-  runPnpmSync(['exec', 'electron-rebuild', '-f', '--', '--msvs_version=2022'], {
+  // Let electron-rebuild delegate Visual Studio discovery to node-gyp so
+  // Windows developers can build with whichever supported MSVC toolchain is
+  // installed, including Visual Studio 2026. The root package.json pins
+  // electron-rebuild to a node-gyp release that supports newer VS versions.
+  runPnpmSync(['exec', 'electron-rebuild', '-f'], {
     cwd: desktopRoot,
     env: commandEnv,
   });

--- a/package.json
+++ b/package.json
@@ -43,7 +43,8 @@
   "packageManager": "pnpm@9.15.0",
   "pnpm": {
     "overrides": {
-      "fast-xml-parser": ">=5.3.4"
+      "fast-xml-parser": ">=5.3.4",
+      "@electron/rebuild>node-gyp": "^12.1.0"
     }
   },
   "devDependencies": {

